inc/utils/content.php

```diff
@@ -564,17 +581,19 @@ function get_categories_array($post){
 
  function odm_echo_extras($postID = "") {
  	 $postID = $postID ? $postID : get_the_ID();
-	 if (function_exists('get_post_meta')) :
-		 $get_author = get_post_meta($postID, 'author', true);
-		 $get_localized_author = get_post_meta($postID, 'author_'.odm_language_manager()->get_current_language(), true);
-	   if ($get_author != '' || $get_localized_author != ''):
-	     $news_source_info = '<span class="lsf">&#xE041;</span> ';
-	     if ($get_localized_author != ''):
-	         $news_source_info .= $get_localized_author.'<br />';
-	     else:
-	         $news_source_info .= $get_author.'<br />';
-	     endif;
-	   endif;
+	 if (function_exists('get_post_meta')):
```

Comment:
Looping  @DBishton in, since this relates to editorial. @DBishton can you please take a look at the conversation we are having here and provide your thoughs?